presidio-analyzer/app.py

```diff
@@ -0,0 +1,85 @@
+from presidio_analyzer.presidio_logger import PresidioLogger
+from presidio_analyzer.analyzer_engine import AnalyzerEngine
+from flask import Flask, request
+import json
+import os
+
+DEFAULT_PORT = "3000"
+
+WELCOME_MESSAGE = r"""
+ _______  _______  _______  _______ _________ ______  _________ _______
+(  ____ )(  ____ )(  ____ \(  ____ \\__   __/(  __  \ \__   __/(  ___  )
+| (    )|| (    )|| (    \/| (    \/   ) (   | (  \  )   ) (   | (   ) |
+| (____)|| (____)|| (__    | (_____    | |   | |   ) |   | |   | |   | |
+|  _____)|     __)|  __)   (_____  )   | |   | |   | |   | |   | |   | |
+| (      | (\ (   | (            ) |   | |   | |   ) |   | |   | |   | |
+| )      | ) \ \__| (____/\/\____) |___) (___| (__/  )___) (___| (___) |
+|/       |/   \__/(_______/\_______)\_______/(______/ \_______/(_______)
+"""
+
+
+class Server:
+
+    def __init__(self):
+        self.logger = PresidioLogger(os.environ.get("PRESIDIO_LOGGER"))
+        self.app = Flask(__name__)
+        self.logger.info("Starting analyzer engine")
+        self.engine = AnalyzerEngine()
+        self.logger.info(WELCOME_MESSAGE)
+
+        @self.app.route('/health')
+        def health() -> str:
+            """
+            Basic health probe.  get ok + 200
+            """
+            return 'ok'
+
+        @self.app.route("/analyze", methods=["POST"])
+        def analyze():
+            """
+            Executes the analyzer function
+            """
+            # Parse the request params
+            req_data = request.get_json()
+
+            text = req_data.get("text")
```

Comment:
I would avoid breaking the json here, and pass the request to a seperate module that will handle it. makes the routes more readable

Reply:
Done